lib/score_card.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+class ScoreCard
+  attr_reader :frames
+
+  def initialize
+    @frames = []
+    @score = 0
+  end
+
+  def play(user_input)
+    final_frame ? tenth_round(user_input) : roll(user_input)
```

Comment:
I like this ternary operation for determining what type of turn this is - clear and efficient!